daml-assistant/get-daml.sh

```diff
@@ -18,16 +18,43 @@
 
 set -eu
 readonly SWD="$PWD"
-readonly TMPDIR="$(mktemp -d)"
+readonly INSTALL_MINSIZE=1000000
+if [ -z $TEMPDIR ]; then
+  readonly TMPDIR="$(mktemp -d)"
+else
+  readonly TMPDIR=$TEMPDIR
```

Comment:
I'm somewhat afraid that `TMPDIR` and `TEMPDIR` are too easy to mix up. Maybe it worth renaming the former into something like `ACTUAL_TEMPDIR`?

Reply:
Also, would it make sense to define `rmTmpDir` in both branches instead of branching in `rmTmpDir` itself? This way, we would have the code that depends on whether `TEMPDIR` is set or not next to each other.